packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -283,6 +283,30 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
     this._onKeyDown(this._onSelectNextMonth, ev);
   }
 
+  private _formatMonthYear(navigatedDate: Date) {
+    let { strings, formatDate } = this.props;
+    let formatedDate;
+
+    if (formatDate && formatDate.formatMonthYear) {
+      formatedDate = formatDate.formatMonthYear(navigatedDate);
+    } else {
+      formatedDate = strings.months[navigatedDate.getMonth()] + ' ' + navigatedDate.getFullYear();
+    }
+    return formatedDate;
+  }
```

Comment:
Instead of having the checks for whether formatDate && formatDate.formatMonthYear, etc being undefined, why dont you use defaultProps to define defaults for all these funcs?